generators/app/templates/common/Makefile

```diff
@@ -14,13 +14,13 @@ install:
 test:
 	poetry run pytest tests --cov src --cov-report term --cov-report=html --cov-report xml --junit-xml=tests-results.xml
 
-black:
-	poetry run black . --check
+ruff-format:
+	poetry run ruff format --check .
 
-ruff:
+ruff-check:
 	poetry run ruff check src tests
 
-fix-ruff:
+ruff-fix:
```

Comment:
To avoid any confusion, I suggest this naming:
- ruff-format-check
- ruff-lint-check
- ruff-lint-fix

And to have more coherence we can add a `ruff-format-fix` command (even if it will mostly be done through IDE integration)

Reply:
Maybe these don't need to mention ruff?

- format-check
- lint-check
- lint-fix

Reply:
You're right, ruff is not necessary in the naming, and it will make autocomplete easier.
Btw, if we follow the same naming logic, we should have `make type-check` instead than `make mypy`
So we have:
- format-check
- format-fix
- lint-check
- lint-fix
- type-check

OK for you?